src/util/openfaas/index.js

```diff
@@ -27,6 +27,14 @@ const FAAS_LIMITS_MEMORY = process.env.FAAS_LIMITS_MEMORY || FAAS_REQUESTS_MEMOR
 const FAAS_MAX_INFLIGHT = process.env.FAAS_MAX_INFLIGHT || '4';
 const FAAS_EXEC_TIMEOUT = process.env.FAAS_EXEC_TIMEOUT || '4s';
 const FAAS_ENABLE_WATCHDOG_ENV_VARS = process.env.FAAS_ENABLE_WATCHDOG_ENV_VARS || 'true';
+const FAAS_READINESS_HTTP_PATH = process.env.FAAS_READINESS_HTTP_PATH || '/ready';
+const FAAS_READINESS_HTTP_INITIAL_DELAY_S = process.env.FAAS_READINESS_HTTP_INITIAL_DELAY_S || '1';
```

Comment:
should we have intial delay as 5sec. given pod needs to load transformations and libraries

Reply:
For smaller transformations 1sec would be enough so they won't need to wait for the probe to start for so long

Reply:
I can increase the threshold of failure to 5 maybe so all in all maybe 11s to download the libraries and transformations